caf-audit-monkey-container/pom.xml

```diff
@@ -105,9 +99,6 @@
                                 </env>
                                 <entryPoint>
                                     <exec>
-                                        <args>/maven/tini-${tini.version}.exe</args>
-                                        <args>-v</args>
-                                        <args>--</args>
                                         <args>/maven/start.sh</args>
                                     </exec>
                                 </entryPoint>
```

Comment:
Either this `entrypoint` element needs to be changed to a `cmd` element, or the arguments specified in the base image need to be duplicated into it.  At the minute it is overwriting the base image entry-point so Tini won't be used and the startup scripts won't run.

Reply:
Apologies, stupid miss this one. Changed `entrypoint` to `cmd` like in other container projects.